src/Docker/Runner/DataLoader/NullDataLoader.php

```diff
@@ -26,7 +26,9 @@ public function loadInputData(
         InputTableStateList $inputTableStateList,
         InputFileStateList $inputFileStateList
     ) {
-        return new StorageState(new Result(), new InputFileStateList([]));
+        $result = new Result();
+        $result->setInputTableStateList(new InputTableStateList([]));
```

Comment:
To mi predim uniklo pri review 🤦‍♂️

Reply:
Mě taky no, navíc už si trochu zvykám, že věci tohoto typu odchytí PhpStan